src/dbworker.cpp

```diff
@@ -580,10 +579,6 @@ void DBWorker::clearHistory()
 
     QList<Link> linkList;
     emit historyAvailable(linkList);
-    if (oldTabCount != 0) {
-        QList<Tab> tabList;
-        emit tabsAvailable(tabList);
-    }
```

Comment:
These lines are redundant as tabsAvailable() gets emitted by this moment inside removeAllTabs() which is called in the line 576